proto/ibc/applications/interchain_accounts/host/v1/tx.proto

```diff
@@ -0,0 +1,28 @@
+syntax = "proto3";
+
+package ibc.applications.interchain_accounts.host.v1;
+
+option go_package = "github.com/cosmos/ibc-go/v7/modules/apps/27-interchain-accounts/host/types";
+
+import "gogoproto/gogo.proto";
+import "ibc/applications/interchain_accounts/host/v1/host.proto";
+
+// Msg defines the 27-interchain-accounts/host Msg service.
+service Msg {
+  // UpdateParams defines a rpc handler for MsgUpdateParams.
+  rpc UpdateParams(MsgUpdateParams) returns (MsgUpdateParamsResponse);
+}
+
+// MsgUpdateParams defines the payload for Msg/UpdateParams
+message MsgUpdateParams {
```

Comment:
ah, also adding the message signer option or else we'll totally miss it after https://github.com/cosmos/ibc-go/pull/3627

Reply:
Shouldn't their PRs take care of these when their PR is merging. If I implement it now, it can be confusing. Even if they have to manually resolve the merge conflict. I think it should be fine?